test/images/volumes-tester/rbd/Dockerfile

```diff
@@ -15,10 +15,10 @@
 # CEPH all in one
 # Based on image by Ricardo Rocha, ricardo@catalyst.net.nz
 
-FROM fedora
```

Comment:
Can you please stick to Fedora? because I have a plan to convert these images to multi-arch like arm, ppc64le etc. So I don't think we really have all these packages available for other archs. As packages are easily available for archs in Fedora, so please shift to latest fedora instead of moving to centos based.

Reply:
Fedora moves forward too quickly. For example, current test images come from Fedora 22 and nobody has touched them for 2 years. It's not possible to get reproducible build of gcr.io/google_containers/volume-rbd, because "FROM fedora:22" will get you a very different image than the one pushed in gcr.io.

I'd like to stick to something more stable if possible

Reply:
The only concern I've is that the packages what we are installing part of centos are not available for other archs like arm, ppc64le. So I'm good if we are able to build this image on all the architecture. @detiber, @luxas @ixdy

Reply:
@mkumatag @jsafrane: CentOS currently only publishes images for amd64 and armv8. While it would be possible to use a different image for ppc64le and s390x, I think that would have more of a potential to cause confusion and lead to arch-specific bugs/edge cases.

I know @jperrin has been working on adding ppc64le support to the official CentOS image, but that would still leave s390x as the odd arch out.

Reply:
Ok, I updated the images to Fedora 26 and tested them.